### PR TITLE
translate 2015-01-01-new-years-2015

### PR DESCRIPTION
**Pull request by aprilnov**

### CODE REVIEW

**Comment by aprilnov** [comment]
Translate 2015-01-01-new-years-2015 done, please review. @croath

**Comment by croath** [comment]
:+1: 吓到我了

**Comment by aprilnov** [comment]
这两天比较有空.. 周末歇两天，周一再来领新的。Happy weekend~

**Comment by croath** [comment]
周末愉快。

**Comment by aprilnov** [comment]
@croath any comments?

File: 2015-01-01-new-years-2015.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+imageView.image = UIImage(contentsOfFile: pathToFileInSharedSubfolder("image.png"))
+```
+
+* * *
+
+The rest of this year's reader submissions come from [Cédric Luthi](https://github.com/0xced), who (as in [years](http://nshipster.com/new-years-2014/) [past](http://nshipster.com/reader-submissions-new-years-2013/)) contributed a payload of tips and tricks worthy of an entire article unto themselves. Thanks so much for these, Cédric!
+
+今年其余的读者意见的来自 [Cédric Luthi](https://github.com/0xced)，他（像[去年](http://nshipster.com/new-years-2014/)或[之前](http://nshipster.com/reader-submissions-new-years-2013/)一样）贡献了很多的技巧和窍门值得占据一整篇文章。非常感谢，Cédric！
+
+## CocoaPods, Exposed!
```
Comment (croath) [inline 2015-01-01-new-years-2015.md]: 这行没有翻译。。。

**Comment by croath** [comment]
[LGTM](http://www.lgtm.in/i/csVQNCm6q)

**Comment by aprilnov** [comment]
@croath I think it's ready for publish, please check, thanks.

**Comment by croath** [comment]
Great job @aprilnov .

**Comment by natecook1000** [comment]
Thanks, @aprilnov & @croath! :clap: :clap:

**Pull request merged**